### PR TITLE
Remove IE6 specific styles

### PR DESCRIPTION
**Pull request by MuriloDalRi**
https://trello.com/c/06BhH9ow/262-confirm-which-browsers-we-are-supporting-and-document

### CODE REVIEW

**Comment by idavidmcdonald** [comment]
https://github.com/alphagov/digitalmarketplace-frontend-toolkit/blob/434ad307913651ecb041ab94bdee748ebe066d1a/toolkit/scss/README.md

I'm looking at the scss readme which looks like it could also get updated. IE6 is mentioned and there is a broken link at the moment.

Just having a review to see if I can find any other changes no longer needed.

**Comment by idavidmcdonald** [comment]
https://github.com/alphagov/digitalmarketplace-frontend-toolkit/blob/a1693c7e0ffabe159d1d5cfaed38bf82920ef2b6/toolkit/scss/forms/_summary.scss#L63

This is also something worth looking into if we can remove it or not (at a glance it looks like yes but I haven't taken the time to fully understand it to make sure it is).

**Comment by idavidmcdonald** [comment]
https://github.com/alphagov/digitalmarketplace-frontend-toolkit/blob/36cd6177319c6b16473cc01c130f4206c159cbab/toolkit/scss/forms/_pricing.scss#L41

**Comment by MuriloDalRi** [comment]
I didn't know about that README file! That should really be part of the root dir one

**Comment by idavidmcdonald** [comment]
Yeah I think I ran into problems with that before - I don't know if I like nested READMEs.

Feel free to move it to the main one and tidy up if you'd like.

**Comment by katstevens** [comment]
Definitely 👎 to nested READMEs - no-one will ever find them.

**Comment by MuriloDalRi** [comment]
There's a link to that file on the main README so I guess it's not _that_ bad

File: toolkit/scss/README.md [hunk 1]
```diff
@@ -27,14 +27,14 @@ stylesheets are wrapped in conditional comments to ensure that:
...
+The SCSS file for IE7 just includes `forms/index.scss` while setting a
 few variables. The mixins from `_conditionals.scss` use these variables
 to determine whether or not to insert the block of browser-specific CSS.
 
 ```
 $is-ie: true;
-$ie-version: 6;
-$mobile-ie6: false;
+$ie-version: 7;
+$mobile-ie7: false;
```
Comment (idavidmcdonald) [inline toolkit/scss/README.md]: This is worth double checking as it might not actually be a flag.

**Comment by samuelhwilliams** [comment]
@MuriloDalRi if this is still good, let's get it in. 👍

**Pull request merged**